vault/resource_kubernetes_secret_backend.go

```diff
@@ -22,7 +22,7 @@ const (
 
 func kubernetesSecretBackendResource() *schema.Resource {
 	resource := &schema.Resource{
-		CreateContext: MinVersionCheckWrapper(kubernetesSecretBackendCreateUpdate, consts.VaultVersion11),
+		CreateContext: MinVersionCheckWrapper(kubernetesSecretBackendCreateUpdate, consts.VaultVersion111),
```

Comment:
Looks like `MinVersionCheckWrapper()` should probably be updated to take a `version.Version` instead of a version string?